frontend/src/main/utils/minMax_NoStryker.js

```diff
@@ -0,0 +1,4 @@
+const arrayMin = (arr) => arr.reduce((p, v) => (p <= v ? p : v));
+const arrayMax = (arr) => arr.reduce((p, v) => (p >= v ? p : v));
+
+export { arrayMin, arrayMax };
```

Comment:
I don't disagree.  I looked far and wide for a recommendation for such a library before implementing this myself--I had trouble finding one.   

FYI: We are going ahead with this in ucsb-cs156-s21 team02 though, because there just isn't time to refactor all of the code at this point.